debezium-core/src/main/java/io/debezium/transforms/outbox/EventRouter.java

```diff
@@ -56,6 +56,7 @@
     private String fieldSchemaVersion;
 
     private String routeByField;
+    private boolean routeTombstoneOnEmptyPayload;
```

Comment:
Now that I'm reading it better, maybe we want to use the word null instead of empty? Since `{} ` can also mean empty. Wdyt?

Reply:
Yeah that’s a good a point. I think that will make the behavior clearer.